extras/index.html

```diff
@@ -26,7 +26,7 @@
       </div>
     </div>
     <div class="section1">
-      <hr style="border: none; height: 1.5rem" />
+      <!-- <hr style="border: none; height: 1.5rem" /> -->
```

Comment:
If you've throughly tested it across multiple browsers and it works to fix the bug. You don't need to comment any of these out, and can just go ahead and delete them.

Reply:
Any, being the ones that you commented out across all files of course.

Reply:
Oh yeah. I commented them because that was the way I deleted them. I can take them away

Reply:
All good.